Scoop/Controller/LoginViewController.swift

```diff
@@ -120,10 +111,22 @@ class LoginViewController: UIViewController {
             case.success(let user):
                 NetworkManager.shared.currentUser = user
             case.failure(let error):
-                print(error.localizedDescription)
+                guard let window = UIApplication.shared.windows.first else {
+                    return
+                }
+                
+                NotificationCenter.default.post(name: NSNotification.Name("didCompleteLogin"), object: nil)
+                if GIDSignIn.sharedInstance.hasPreviousSignIn() {
+                    GIDSignIn.sharedInstance.signOut()
+                    self.presentErrorAlert(title: "Failed to Login", message: "Please make sure you have a stable internet connection and try again later")
+                } else {
+                    let onboardingVC = OnboardingContainerViewController()
+                    onboardingVC.modalPresentationStyle = .fullScreen
+                    window.rootViewController?.present(onboardingVC, animated: false)
+                    print(error.localizedDescription)
```

Comment:
might want to add a description for the error before printing the error here

Reply:
I would also print out this error outside of the if-else